docs/configuration.asciidoc

```diff
@@ -0,0 +1,28 @@
+[[configuration]]
+== Configuration
+
+To adapt the Elastic APM agent to your needs, you can configure it using environment variables.
+
+
+[[config-reporter]]
+=== Reporter configuration options
+[float]
+[[config-server-urls]]
+==== `server_urls`
+
+[options="header"]
+|============
+| Environment variable name
+| `ELASTIC_APM_SERVER_URLS`
+|============
+
+[options="header"]
+|============
+| Default                 | Type
+| `http://localhost:8200` | List
+|============
+
+The URLs for your APM Servers. The URLs must be fully qualified, including protocol (`http` or `https`) and port. To add multiple servers separate them with comma (`,`).
```

Comment:
> The URLs for your APM Servers.

Is this intended to be a title/heading?

Reply:
No. Should it be? It's stolen from the [Python doc](https://github.com/elastic/apm-agent-python/blob/master/docs/configuration.asciidoc).

Reply:
My mistake, this is okay.